app/common/directives/mainsheet-container.html

```diff
@@ -0,0 +1,9 @@
+<div class="mainsheet" ng-show="showMainsheet" style="display: block;">
```

Comment:
add the classes ```init active``` after `mainsheet`. And ```opacity: 1;``` to style... And also change to ng-style if possible, uglify often mess up the 'normal' inline-styles.